### ISSUE TITLE
Pipeline Referencing missing language and version switchers

### ISSUE THREAD

**Issue by rgarrity** [jen-jackson/l10n-arcmap-certification#3]
* Language: English and others
* URL: http://desktopstg.arcgis.com/en/arcmap/latest/extensions/pipeline-referencing/get-started-with-arcgis-pipeline-referencing.htm
* Topic title (EN): Get started with pipeline referencing
* Topic GUID: [GUID-A5839CEB-B5CC-4E88-ACE3-54408C548528](https://devtopia.esri.com/ArcGISPro/help-feedback/search?type=Issues&utf8=✓&q=GUID-A5839CEB-B5CC-4E88-ACE3-54408C548528)
* Publication GUID: GUID-82076303-7CA3-40C8-94EC-FB5F096404FB
* Screenshot:
![image](https://cloud.githubusercontent.com/assets/7049342/23227244/26f44454-f8ed-11e6-9bac-2c4c41f53459.png)

Issue description:
Topics in this publication are missing the language switcher and a version switcher.

**Comment by jen-jackson** [comment]
Asking @igalvan about this. Based on the web UI, it appears that 10.5 is the first time this doc was published to the web.

**Comment by igalvan** [comment]
Correct, 10.5 is the first release, so the version switcher should not be there

**Comment by michi6571** [comment]
@rgarrity Thank you for the report.  

Aimee helped us to add the language switcher.  But, it required to republish the pub to get the switcher.  The localized publication has been republished to DEV, UAT, and STG.  
@igalvan Please publish English one to get the switcher.

**Comment by igalvan** [comment]
published, but also noticed that the Location Referencing toolbox is also missing the language switcher, should I republish that right now too or wait for some additional work? http://desktopdev.arcgis.com/en/arcmap/latest/tools/location-referencing-pipeline-toolbox/an-overview-of-the-location-referencing-toolbox.htm

**Comment by igalvan** [comment]
On further discussions with El and Aimee (see issues https://github.com/ArcGIS/arcgis-for-desktop-website/issues/967 and https://github.com/ArcGIS/arcgis-for-desktop-website/issues/1074) new pubs should include the version switcher, but have previous versions disabled

**Comment by igalvan** [comment]
Republished the toolbox pub and it now includes the language switcher

**Comment by michi6571** [comment]
Thank you, @igalvan 
The localized pubs also republished and have the language switcher.
